notebook/automl_nlp.ipynb

```diff
@@ -631,15 +461,102 @@
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/ipykernel/kernelbase.py\", line 473, in dispatch_queue\n",
+      "    await self.process_one()\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/ipykernel/kernelbase.py\", line 462, in process_one\n",
+      "    await dispatch(*args)\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/ipykernel/kernelbase.py\", line 369, in dispatch_shell\n",
+      "    await result\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/ipykernel/kernelbase.py\", line 664, in execute_request\n",
+      "    reply_content = await reply_content\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/ipykernel/ipkernel.py\", line 355, in do_execute\n",
+      "    res = shell.run_cell(code, store_history=store_history, silent=silent)\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/ipykernel/zmqshell.py\", line 532, in run_cell\n",
+      "    return super().run_cell(*args, **kwargs)\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/IPython/core/interactiveshell.py\", line 2854, in run_cell\n",
+      "    result = self._run_cell(\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/IPython/core/interactiveshell.py\", line 2900, in _run_cell\n",
+      "    return runner(coro)\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/IPython/core/async_helpers.py\", line 129, in _pseudo_sync_runner\n",
+      "    coro.send(None)\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/IPython/core/interactiveshell.py\", line 3098, in run_cell_async\n",
+      "    has_raised = await self.run_ast_nodes(code_ast.body, cell_name,\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/IPython/core/interactiveshell.py\", line 3301, in run_ast_nodes\n",
+      "    if await self.run_code(code, result, async_=asy):\n",
+      "  File \"/data/installation/anaconda3/envs/tmp/lib/python3.8/site-packages/IPython/core/interactiveshell.py\", line 3361, in run_code\n",
+      "    exec(code_obj, self.user_global_ns, self.user_ns)\n",
+      "  File \"/tmp/ipykernel_49710/1077267443.py\", line 8, in <cell line: 4>\n",
+      "    automl.trainable(config)\n",
+      "  File \"/data/xliu127/projects/hyperopt/FLAML/flaml/automl.py\", line 1700, in train\n",
+      "    \"metric_for_logging\": np.inf,\n",
+      "  File \"/data/xliu127/projects/hyperopt/FLAML/flaml/automl.py\", line 299, in _compute_with_config_base\n",
+      "    tune.report(**result)\n",
+      "  File \"/data/xliu127/projects/hyperopt/FLAML/flaml/tune/tune.py\", line 88, in report\n",
+      "    return tune.report(_metric, **kwargs)\n",
+      "\n"
+     ]
+    }
```

Comment:
Isn't the error gone?

Reply:
Sorry, I did sync the notebook from my server to local, I don't know why it wasn't updated. I updated it again. 

Added item 7 to the todo list: https://docs.google.com/document/d/1iKaywdGlg_Rc2Ri7qGiXqygvus-BNVWEzX0GKR7n2-Q/edit?usp=sharing